web/src/message_list_hover.js

```diff
@@ -42,7 +46,7 @@ export function message_hover($message_row) {
 
     const can_move_message = message_edit.can_move_message(message);
     const args = {
-        is_content_editable: is_content_editable && !message.status_message,
+        is_content_editable: is_content_editable && status_message,
```

Comment:
This is super confusing. First, you've negated the meaning. But also, we just need the boolean state, and that's available as  `message.is_me_message`.

Reply:
Yeah. Apologies for the negligence. I have fixed it now.